modules/user/client-react/index.native.jsx

```diff
@@ -135,6 +136,15 @@ ProfileScreen.propTypes = {
   navigation: PropTypes.object
 };
 
+class ActivateUserScreen extends React.Component {
```

Comment:
Can you use Function component instead of Class component?

Reply:
Done